origami.py

```diff
@@ -761,8 +761,10 @@ def on_close(self, view):
 
     def on_pre_close(self, view):
         # Read from global settings for backward compatibility
-        auto_close = view.settings().get("origami_auto_close_empty_panes", False)
-        auto_close = self.settings().get("auto_close_empty_panes", auto_close)
+        auto_close = (
+            view.settings().get("origami_auto_close_empty_panes")
+            or self.settings().get("auto_close_empty_panes")
+        )
```

Comment:
We've created a fix at the same time (#167) :)

Your handles a setting that I didn't but the logic is not correct, I think. We want to use `origami_auto_close_empty_panes` if it's a boolean, even if it is false.

Reply:
:LOL: That was the lazy approach.  We can have a `is None` check of course.  Wait a sec.